aiida_quantumespresso/workflows/q2r/base.py

```diff
@@ -21,46 +19,46 @@ class Q2rBaseWorkChain(BaseRestartWorkChain):
 
     @classmethod
     def define(cls, spec):
+        # yapf: disable
         super(Q2rBaseWorkChain, cls).define(spec)
-        spec.input('code', valid_type=orm.Code)
-        spec.input('parent_folder', valid_type=orm.FolderData)
-        spec.input('parameters', valid_type=orm.Dict, required=False)
-        spec.input('settings', valid_type=orm.Dict, required=False)
-        spec.input('options', valid_type=orm.Dict, required=False)
+        spec.expose_inputs(Q2rCalculation, namespace='q2r')
+        spec.expose_outputs(Q2rCalculation, exclude=('remote_folder',))
         spec.outline(
             cls.setup,
-            cls.validate_inputs,
             while_(cls.should_run_calculation)(
                 cls.run_calculation,
                 cls.inspect_calculation,
             ),
             cls.results,
         )
-        spec.output('force_constants', valid_type=ForceconstantsData)
```

Comment:
This becomes `forceconstants` (no underscore). Just checking that it's what you want.

Reply:
I actually prefer it with the underscore but since the `Q2rCalculation` for now uses without, I opted for consistency. I still think it would be good to change it, but then for both, before the release of `3.0.0`

Reply:
I agree. I [made an issue](https://github.com/aiidateam/aiida-quantumespresso/issues/383) so we don't forget, but if you want to do it in this PR it's ok for me.

Reply:
Thanks, let's do that one in a separate PR since it might affect other things